doc/ADMIN.md

```diff
@@ -0,0 +1,4 @@
+* Email required for administation (uses "magic links" instead of admin passwords)
+
+* You should manually configure the app after installation.
+* You should edit __INSTALL_DIR__/run_fider to add environment variables, e.g for GitHub integration
```

Comment:
I'm confused between this line and what's in POST_INSTALL.md, which mentions `__INSTALL_DIR__/custom_fider_env ` (is it the same file than `run_fider` or a different one ?)

I don't know much about this app but it sounds like the whole thing about Github thing should be moved from POST_INSTALL.md to here

Reply:
> __INSTALL_DIR__/custom_fider_env (is it the same file than run_fider or a different one ?)

custom_fider_env is a custom file, that allows custom config, and will not be overwritten by the package updates